libvast/src/format/csv.cpp

```diff
@@ -293,6 +293,9 @@ struct csv_parser_factory {
 
   template <class T>
   result_type operator()(const T& t) const {
+    [[maybe_unused]] const auto field
+      = parsers::qqstr
+        | +(parsers::any - opt_.set_separator - opt_.kvp_separator);
```

Comment:
Why are we excluding `set_separator` and `kvp_separator` here, as opposed to just `separator` which was excluded in the ad-hoc parsers before?

Reply:
Good spot. It's a leftover from debugging, will fix.

Reply:
Fixed.